lib/rules/no-multi-spaces.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @fileoverview This rule warns about the usage of extra whitespaces between attributes
+ * @author Armano
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'This rule warns about the usage of extra whitespaces between attributes',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: 'whitespace',  // or "code" or "whitespace"
+    schema: []
+  },
+
+  /**
+   * @param {RuleContext} context - The rule context.
+   * @returns {Object} AST event handlers.
+   */
+  create (context) {
+    const sourceCode = context.getSourceCode()
+
+    // ----------------------------------------------------------------------
+    // Public
+    // ----------------------------------------------------------------------
+
+    return {
+      Program (node) {
+        // TODO: Check `context.parserServices.getTemplateBodyTokenStore` exists or not.
+        const tokenStore = context.parserServices.getTemplateBodyTokenStore()
+        const tokens = tokenStore.getTokens(node.templateBody, { includeComments: true })
+
+        let prevToken = tokens.shift()
+        for (const token of tokens) {
+          if (sourceCode.isSpaceBetweenTokens(prevToken, token)) {
```

Comment:
This method cannot be used here because this method ignores `/* */` style comments which are not comments in HTML context.